.gitignore

```diff
@@ -1,2 +1,3 @@
 plugins/
 vendor/
+.idea
```

Comment:
I would add even more there, at least composer.lock. Also when you run tests they generate another bunch of files that are asking to be commited.